### DISCUSSION TITLE
Include pop-up menu of alias?

### DISCUSSION THREAD

**Discussion by rwijtvliet** [FelixKratz/SketchyBar#398] [Q&A]
I can't find the answer to this question, though it's probably come up before.

With `sketchybar --add alias "Control Centre,Bluetooth" right` I can add the bluetooth icon to sketchybar.

**Is there a way to also have the same pop-up menu on click?** i.e., where I can turn bluetooth on/off, see connected devices, and go to the bluetooth menu?

Same goes for wifi, datetime, power, etc.

**Comment by mcjkula** [comment]
Got the same question, couldn't find an answer anywhere.

**Comment by rwijtvliet** [comment]
Ah, I have found the answer in the meantime. I can't find it right now, but the TLDR was that it's tricky and that you shouldn't hold your breath ;)

**Reply by mcjkula** [reply]
Oh nice, if you would be able to find it again, I would appreciate it. 🙏
